jwst/lib/tests/test_pipe_utils.py

```diff
@@ -47,6 +48,20 @@ def test_is_tso_from_tsoflag(tsovisit, expected):
     assert pipe_utils.is_tso(model) is expected
 
 
+def test_is_tso_nrcgrism_nints1():
+    """Test is_tso with NRC_TSGRISM and NINTS=1"""
+    model = datamodels.JwstDataModel()
+    model.meta.exposure.type = "NRC_TSGRISM"
+    model.meta.visit.tsovisit = False
+    model.meta.exposure.nints = 10
+    assert pipe_utils.is_tso(model) is True
+
+    model.meta.exposure.nints = 1
+    assert pipe_utils.is_tso(model) is False
+
+    assert (is_tso(model) or model.meta.exposure.type.lower() in dms_base.TSO_EXP_TYPES) is True
```

Comment:
You don't need to `assert is True`, because the assertion itself returns `True` or `False`.

Reply:
done